content/src/service/deployments/DeploymentManager.ts

```diff
@@ -159,9 +159,17 @@ export type PartialDeploymentHistory = {
     },
 }
 
-export type DeploymentFilters = {
-    fromLocalTimestamp?: Timestamp
-    toLocalTimestamp?: Timestamp
+export type  DeploymentFilters = {
+    fromLocalTimestamp?: Timestamp,
+    toLocalTimestamp?: Timestamp,
+    fromOriginTimestamp?: Timestamp,
+    toOriginTimestamp?: Timestamp,
+    originServerUrl?: ServerAddress,
```

Comment:
Maybe this should be an array too.

Reply:
This is currently not exposed. It is there so the `/history` endpoint can use it. If we decide to expose it, then we should discuss how to do so